src/libvirt-xml-parse.js

```diff
@@ -481,6 +483,21 @@ export function parseDumpxmlForVsock(devicesElem) {
     return { cid };
 }
 
+function parseDumpxmlForSpice(devicesElem) {
+    for (let i = 0; i < devicesElem.children.length; ++i) {
+        const device = devicesElem.children.item(i);
+        // also catch spicevmc
+        if (device.getAttribute("type")?.startsWith("spice"))
+            return true;
+        // qxl video is also related to SPICE
+        if (device.tagName === "video" && getSingleOptionalElem(device, "model")?.getAttribute("type") === "qxl")
+            return true;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1387-20240112-155815-cbcdc423-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)